source/core/src/main/com/csse3200/game/components/ship/ShipAnimationController.java

```diff
@@ -36,27 +36,27 @@ private void animateShipStage(int progress, Set<Feature> unlockedFeatures) {
 		String animation = "default";
 
 		//TODO change states to match final amount of progress
```

Comment:
code smell: if this is done can you please remove, if not ignore

Reply:
well just in case we change anything before tues :)